### PR TITLE
Disable log printing GPU and set GCS output using `--env`

### PR DESCRIPTION
**Pull request by jonb377**
# Description

GPU tests were broken by the last change, which enables GCS output path and log printing.

This change will depend on https://github.com/google/xpk/pull/104 to add `--env` support in XPK.

# Tests

Ran a local XPK test (did not have permissions to test in the GPU project):
- https://screenshot.googleplex.com/Kmw4s83FgKnXUBx

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: xlml/utils/xpk.py
Comment (NinaCai) [inline xlml/utils/xpk.py:72]: Does xpk has `--env` argument?

Comment (jonb377) [reply xlml/utils/xpk.py:72]: I just added in https://github.com/google/xpk/pull/104, we'll have to wait for that one to land before this can merge.

File: xlml/utils/xpk.py
Comment (RissyRan) [inline xlml/utils/xpk.py:144]: logs from 1 pod works fine right? I don't have source to the local run.

**Review by RissyRan — APPROVED** [review]
LGTM, just one question to check.

File: xlml/utils/xpk.py
Comment (jonb377) [reply xlml/utils/xpk.py:144]: Ah yeah, single-container jobs work: https://screenshot.googleplex.com/5JcEKeSeBBSohho

**Comment by jonb377** [comment]
Thanks Ran!

**Pull request merged**
